### DISCUSSION TITLE
the problem for creating a model with local file  in an free account

### DISCUSSION THREAD

**Discussion by dongluyang** [iTwin/itwinjs-core#4386] [General]
After i created a model with option of local file in my free account, I found its result was empty in view connect.
what is wrong with my operation. is it because my account is free? But I found the price list show free account has 1 GB storage.
anyone can help me. thank you sincerely in advance. the below is my model file of dgn format
[20201024fenlei.dgn.zip](https://github.com/iTwin/itwinjs-core/files/9663779/20201024fenlei.dgn.zip)

**Comment by pmconne** [comment]
Hi @dongluyang,

This has nothing to do with your account. The DGN file you provided contains no elements.
